tests/routes/test_cog.py

```diff
@@ -326,3 +329,5 @@ def test_tile_outside_bounds_error(reader, app):
     reader.side_effect = mock_reader
     response = app.get("/cog/tiles/15/0/0?url=https://myurl.com/cog.tif&rescale=0,1000")
     assert response.status_code == 404
+    # NOT THIS MIGHT CHANGE
+    assert response.headers["Cache-Control"] == "public, max-age=3600"
```

Comment:
Should we add `Cache-Control` to responses with 4** and 5** status code?

Reply:
I think it makes sense to include for 4XX but not 5XX

Reply:
5XX is server error right? Definitely shouldn't cache those responses...

Reply:
cool, gonna skip cache for those